openedx/core/djangoapps/catalog/utils.py

```diff
@@ -162,10 +162,10 @@ def munge_catalog_program(catalog_program):
             } for course in catalog_program['courses']
         ],
         'banner_image_urls': {
-            'w1440h480': catalog_program['banner_image']['large']['url'],
-            'w726h242': catalog_program['banner_image']['medium']['url'],
-            'w435h145': catalog_program['banner_image']['small']['url'],
-            'w348h116': catalog_program['banner_image']['x-small']['url'],
+            'w1440h480': catalog_program.get('banner_image', {}).get('large', {}).get('url'),
```

Comment:
@MatthewPiatetsky true, but those programs shouldn't be coming back to the LMS. Inactive programs are [filtered out](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/catalog/utils.py#L59).